src/main/webapp/app/pages/curation/CurationPage.tsx

```diff
@@ -239,271 +250,333 @@ const CurationPage = (props: ICurationPageProps) => {
     );
   }
 
-  return !!props.data && drugList.length > 0 && !props.loadingGenes ? (
+  const handleReviewButtonClick = () => {
+    if (isReviewing) {
+      setIsReviewFinished(false);
+    }
+    props.updateMeta(`${firebaseMetaPath}`, { review: { currentReviewer: isReviewing ? '' : props.fullName } });
+  };
+
+  const handleReviewFinished = (isFinished: boolean) => {
+    setIsReviewFinished(isFinished);
+  };
+
+  const getReviewButtons = () => {
+    let button;
+    if (geneNeedsReview(props.metaData)) {
+      if (isReviewing || isReviewFinished) {
+        button = (
+          <Button color="primary" onClick={handleReviewButtonClick}>
+            Review Complete
+          </Button>
+        );
+      } else {
+        button = (
+          <Button outline color="primary" onClick={handleReviewButtonClick}>
+            Review
+          </Button>
+        );
+      }
+    } else {
+      if (isReviewFinished) {
+        button = (
+          <Button color="primary" onClick={handleReviewButtonClick}>
+            Review Complete
+          </Button>
+        );
+      } else {
+        return undefined;
```

Comment:
Can this really happen?

Reply:
`isReviewFinished` is false by default (and only gets set in the review mode) and when a gene doesn't need review, this condition will be hit. We don't want to show a button in this scenario.